src/blocks.cpp

```diff
@@ -17,6 +24,11 @@ Block::Block(int row, int col, sf::Color clr) {
 	this->block.setOutlineColor(sf::Color::White);
 }
 
+void Block::operator=(Block& newBlock) {
+	sf::RectangleShape newShape = newBlock.block;
+	block = newShape;
```

Comment:
there are 2 copies of sf::RectangleShape being done here. Just set block to newBlock.block.